.eslintrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
     '*.d.ts',
     'babel-cjs.js',
     'babel-esm.js',
+    'storybook-static'
```

Comment:
Oh, sick. Does this speed things up?

Reply:
ye a fair bit - also caused local failures for me